packages/upload/theme/lumo/vaadin-upload-styles.js

```diff
@@ -24,11 +24,6 @@ registerStyles(
       transition: background-color 0.6s, border-color 0.6s;
     }
 
-    [part='primary-buttons'] > * {
-      display: inline-block;
-      white-space: nowrap;
-    }
```

Comment:
These styles don't seem to be needed anymore, because the `vaadin-button` already has them built-in:

https://github.com/vaadin/web-components/blob/090aff78a3e010aae72d0fe7b7367e462a3d7671/packages/button/src/vaadin-button.js#L56-L59